test/built-ins/AsyncIteratorPrototype/Symbol.asyncIterator/return-val.js

```diff
@@ -10,11 +10,22 @@ features: [Symbol.asyncIterator, async-iteration]
 ---*/
 
 async function* generator() {}
-var AsyncIteratorPrototype = Object.getPrototypeOf(Object.getPrototypeOf(generator.prototype))
+const AsyncIteratorPrototype = Object.getPrototypeOf(Object.getPrototypeOf(generator.prototype))
```

Comment:
I took some liberty in converting the `var` to `const`; is that ok? Would it be preferable to keep them as `var`s or turn them to `let`s or whichever way is fine?

Reply:
I don't think this is a problem either way — personally I prefer `const` in new code but again this is something that we should codify in a linter rule if we really care about it.